addons/website/static/src/components/website_loader/website_loader.scss

```diff
@@ -210,14 +210,14 @@
             animation-play-state: inherit;
         }
         img {
-            width: percentage(1/8);
+            flex-basis: 0 0 percentage(1/8);
```

Comment:
@stefanorigano `flex-basis` is a one-value property, so this is invalid, so this whole rule is ignored... so it seems that is works without needing this at all? :thinking: I let you check :pray:

Reply:
> flex-basis is a one-value property

Oops... When you think about something, but you're typing something else 🥲😬
Sorry, was supposed to be `flex:`, so: 

```diff
- flex-basis: 0 0 percentage(1/8);
+ flex: 0 0 percentage(1/8);
```
Anyway, as you pointed out, the animation works even without those rules, though not exactly as intended.

My initial goal was to achieve seamless looping, so each animation could run indefinitely—just for good practice, and in case we needed to allocate more time to one message. However, since users never reach the end of the loop right now, removing the rules is fine.

Code an commit/PR descriptions updated accordingly, thanks!